lib/Dialect/SV/Transforms/HWMemSimImpl.cpp

```diff
@@ -190,6 +253,73 @@ void HWMemSimImplPass::runOnOperation() {
     if (genOp.descriptor() == "FIRRTL_Memory") {
       auto mem = analyzeMemOp(oldModule);
 
+      // If this memory has defined write-after-write behavior, we need to
+      // determine which sets of write ports are driven by the same clock.
+      if (mem.writeUnderWrite == 1) {
+
+        auto symbol = SymbolRefAttr::get(oldModule.getNameAttr());
+        auto instances = symbolDigger.get(symbol);
+        bool differentClocks = false;
+        auto buildWriteSets = [&](Operation *op) {
```

Comment:
Please add a doc commend describing what buildWriteSets does

Reply:
This was removed in the rebase.